src/service/auth-services.jsx

```diff
@@ -2,18 +2,33 @@ import { tokenKey } from "../config";
 import collectionClient from "./collection-client";
 
 export async function login(credentials) {
-	const {token, ...user} = await collectionClient("/login", {
-	body: credentials 
-	});
+  const { ...user } = await collectionClient("/login", {
+    body: credentials,
+  });
 
-	sessionStorage.setItem(tokenKey, token);
-	return user;
+  return user;
 }
 
 export async function logout() {
-	await collectionClient("/logout", {
-		method: "DELETE",
-	});
+  await collectionClient("/logout", {
+    method: "DELETE",
+  });
 
-	sessionStorage.removeItem(tokenKey);
+  sessionStorage.removeItem(tokenKey);
+}
+
+export async function loginRecruiter(credentials) {
+  const { ...user } = await collectionClient("/company/sign_in", {
+    body: credentials,
+  });
+
+  return user;
+}
```

Comment:
I wonder... how will the front end know which user is interacting with the application if they are not receiving a token when logging in? 🤔

Reply:
I already found the answer to my question 😅  [(service/collection-client)](https://github.com/sebastianterleira/get-that-job/pull/2/files#diff-483cadfc701f0f71a036b37179dfddab6101fda00c857c5a3c26b2744e844e43R42-R43)